build.gradle

```diff
@@ -59,7 +59,7 @@ wrapper {
 
 def installFrontend = tasks.register('installFrontend', com.liferay.gradle.plugins.node.tasks.ExecutePackageManagerTask) {
     workingDir 'frontend/'
-    args = ['install']
+    args = ['ci']
```

Comment:
There's an `installCypress` task at line 188 that also needs to have the args changed from `['install']` to `['ci']`

Reply:
Please check your commit message character count. The longest line has 101 characters.

Reply:
Apologies for missing that, I will update that as well.